spec/controllers/hub/sites_controller_spec.rb

```diff
@@ -122,15 +123,32 @@
     context "as an authenticated admin user" do
       before { sign_in admin_user }
 
-      it "updates the site with the new attributes and redirects to the input organization edit page" do
-        post :update, params: params
+      context "when site object is valid" do
```

Comment:
Similar to `organizations_controller_spec.rb`, consider `"with valid params"` and `"with invalid params"`. Not a blocker.